docs/latest/design/egctl.md

```diff
@@ -0,0 +1,64 @@
+# Introduce egctl
+
+## Motivation
+
+EG should provide a command line tool with following capabilities:
+
+- One-stop installation/uninstallation experience
+- Collect configuration form data plane
```

Comment:
can we add another similar bullet for control plane (to collect logs and config)